yt_dlp/extractor/joqrag.py

```diff
@@ -0,0 +1,54 @@
+import urllib.parse
+
+from .common import InfoExtractor
+
+
+class JoqrAgIE(InfoExtractor):
+    IE_DESC = '超!A&G+ 文化放送 Nippon Cultural Broadcasting, Inc. (JOQR)'
+    _VALID_URL = r'''(?x)
+                    (https?://www\.uniqueradio\.jp/agplayer5/player\.php)|
+                    (https?://www\.uniqueradio\.jp/agplayer5/inc-player-hls\.php)|
+                    (https?://(?:www\.)?joqr\.co\.jp/ag/)|
+                    (https?://(?:www\.)?joqr\.co\.jp/qr/(?:agdailyprogram|agregularprogram)/)
+                    '''
+    _TESTS = [{
+        'url': 'https://www.uniqueradio.jp/agplayer5/player.php',
+        'info_dict': {
+            'id': 'live',
+            'title': str,
+            'description': str,
+            'live_status': 'is_live',
+        },
+        'params': {
+            'skip_download': True,
+            'ignore_no_formats_error': True,
+        },
+    }]
+
+    def _real_extract(self, url):
+        video_id = 'live'
+
+        metadata = self._download_webpage(
+            'https://www.uniqueradio.jp/aandg', video_id,
+            note='Downloading metadata', errnote='Failed to download metadata')
+        title = urllib.parse.unquote_plus(
+            self._search_regex(r'var\s+Program_name\s*=\s*["\']([^"\']+)["\']', metadata, 'program title'))
+        desc = urllib.parse.unquote_plus(
+            self._search_regex(r'var\s+Program_text\s*=\s*["\']([^"\']+)["\']', metadata, 'program description'))
```

Comment:
as i write this the description that gets extracted is `<div>土曜の黄昏時にお届けするノンストップアニソンライブラリー</div><div>ぜひ何かをしながらお聴きください！</div><br>番組へのメール、リクエストは<a href=\"mailto:sag@joqr.net\">sag@joqr.net</a>まで！<br>`
```suggestion
        desc = clean_html(urllib.parse.unquote_plus(
            self._search_regex(r'var\s+Program_text\s*=\s*["\']([^"\']+)["\']', metadata, 'program description')))
```

also add ```from ..utils import clean_html``` at the top
probably wouldn't hurt to do the same on the title as well, though that doesn't seem to immediately need it